python/src/robyn/modeling/entities/enums.py

```diff
@@ -0,0 +1,30 @@
+from enum import Enum, auto
+
+
+class NevergradAlgorithm(Enum):
```

Comment:
@gufengzhou do we support all of the algorithms listed below? We will keep list of supported algorithms only here.

Reply:
Seems TWO_POINTS_DE is default but it can take other algorithms.